content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -156,19 +156,12 @@ If you do not specify `.spec.replicas`, then it defaults to 1.
 ### Deleting a ReplicaSet and its Pods
 
 To delete a ReplicaSet and all its pods, use [`kubectl
-delete`](/docs/reference/generated/kubectl/kubectl-commands#delete). Kubectl will scale the ReplicaSet to zero and wait
-for it to delete each pod before deleting the ReplicaSet itself. If this kubectl command is interrupted, it can
-be restarted.
-
-When using the REST API or go client library, you need to do the steps explicitly (scale replicas to
```

Comment:
note the delete options to use to delete children when using the REST API or client library (`?propagationPolicy=Background` or `?propagationPolicy=Foreground`)

Reply:
Done.